### PR TITLE
Improve the top-level command descriptions even further

### PR DESCRIPTION
**Pull request by danielbachhuber**
See https://github.com/wp-cli/wp-cli/issues/4344

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +13,18 @@ This package implements the following commands:
...
+Manipulate the WP Object Cache object.
 
 ~~~
 wp cache
 ~~~
 
-Use a persistent object cache drop-in to persist cache values between requests.
+By default, the WP Object Cache exists in PHP memory for the length of the
+request (and is emptied at the end). Use a persistent object cache drop-in
+to persist the object cache between request.
```
Comment (schlessera) [inline README.md]: `between request` => `between requests`

File: src/Cache_Command.php [hunk 2]
```diff
@@ -1,11 +1,14 @@
 <?php
 
 /**
- * Manipulates the WP Object Cache.
+ * Manipulate the WP Object Cache object.
  *
+ * By default, the WP Object Cache exists in PHP memory for the length of the
+ * request (and is emptied at the end). Use a persistent object cache drop-in
+ * to persist the object cache between request.
```
Comment (schlessera) [inline src/Cache_Command.php]: `between request` => `between requests`

**Review by schlessera — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (danielbachhuber) [reply README.md]: 9308abe7

File: src/Cache_Command.php (hunk 2 above)
Comment (danielbachhuber) [reply src/Cache_Command.php]: 9308abe7

**Comment by danielbachhuber** [comment]
@schlessera Updated

**Review by schlessera — APPROVED** [review]

**Pull request merged**